servatrice/migrations/servatrice_0017_to_0018.sql

```diff
@@ -0,0 +1,5 @@
+-- Servatrice db migration from version 17 to version 18
+
+alter table cockatrice_users add privlevel enum("NONE","VIP","DONATOR") NOT NULL;
```

Comment:
This actually works both ways (with or without the word column).  At least in the version of mariadb that I am running.  Would you like me to add the word column to the script?